### PR TITLE
Institution info

### PR DESCRIPTION
**Pull request by TheseusDeus**
Added more points to the institution info by request of Karianne

### CODE REVIEW

File: GiEnJul/ClientApp/src/components/institution/InstitutionInfo.tsx [hunk 1]
```diff
@@ -1,42 +1,39 @@
...
-          </Typography>
-        </li>
-        <li>
-          <Typography variant="body1">
-            Husk å skrive dersom familien har spesielle behov.
-          </Typography>
-        </li>
-      </ul>
+const information: string[] = [
+  "Før du melder inn en familie må familien ha sag ja til å bli med, og du har all informasjon som trengs nedenfor.",
```
Comment (Sommerli) [inline GiEnJul/ClientApp/src/components/institution/InstitutionInfo.tsx:5]:
```suggestion
  "Før du melder inn en familie må familien ha sagt ja til å bli med, og du har all informasjon som trengs nedenfor.",
```

**Review by Sommerli — CHANGES_REQUESTED** [review]
En skrivefeil

File: GiEnJul/ClientApp/src/components/institution/InstitutionInfo.tsx [hunk 2]
```diff
@@ -1,42 +1,39 @@
...
-          <Typography variant="body1">
-            Husk å skrive dersom familien har spesielle behov.
-          </Typography>
-        </li>
-      </ul>
+const information: string[] = [
+  "Før du melder inn en familie må familien ha sagt ja til å bli med, og du har all informasjon som trengs nedenfor.",
+  "Dersom du har en ID på familien (som PID), må det legges inn.",
+  "Mange givere synes det er fint å kunne kjøpe inn gaver, og ikke bare gavekort.",
+  "Gaveønskene skal ligge på rundt 300 per person. Kom gjerne med flere ønsker, så har giveren noe å velge mellom.",
```
Comment (joffe97) [inline GiEnJul/ClientApp/src/components/institution/InstitutionInfo.tsx:8]: Should this be `300 kr per person`?

Comment (TheseusDeus) [reply GiEnJul/ClientApp/src/components/institution/InstitutionInfo.tsx:8]: sure

**Review by elisabethnetland — APPROVED** [review]

**Pull request merged**